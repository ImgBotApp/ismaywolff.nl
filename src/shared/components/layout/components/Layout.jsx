import styled from 'styled-components'

const Layout = styled.div`
  display: grid;
  grid-template-rows: 1fr auto;
  min-height: 100vh;
`

export default Layout
