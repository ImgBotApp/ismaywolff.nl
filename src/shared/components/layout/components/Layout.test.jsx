import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import Layout from './Layout'

describe('<Layout />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Layout />)
    expect(wrapper).toMatchSnapshot()
  })
})
