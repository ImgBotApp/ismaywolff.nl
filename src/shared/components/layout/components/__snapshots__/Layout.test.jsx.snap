// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-rows: 1fr auto;
  min-height: 100vh;
}

<div
  className="c0"
/>
`;
